F1WM.IntegrationTests/TracksTests.cs

```diff
@@ -15,12 +17,39 @@ public async Task ShouldGetTrackRecords(TrackRecordsTestData data)
 				data.Expected);
 		}
 
+		[Theory]
+		[JsonData("tracks", "track-records.json")]
+		public async Task ShouldGetTracks(TracksSummaryTestData data)
+		{
+			await TestResponse<TrackSummary>(
+				$"{baseAddress}/tracks?countPerPage={data.CountPerPage}&page={data.Page}",
+				data.Expected);
+		}
+
+		[Theory]
+		[JsonData("tracks", "track-records.json")]
+		public async Task ShouldGetTracksByStatusId(TracksSummaryTestData data)
```

Comment:
You shouldn't use the same kind of data for different test methods. Either merge them into one method (kind of tricky) or create two separate test data files and test data models.